R/R-optmatch/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           R 1.0
+
+R.setup             github markmfredrickson optmatch 0.10.7 v
+revision            0
+categories-append   math
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             GPL-2+
+description         Functions for optimal matching
+long_description    {*}${description}
+checksums           rmd160  86694ffac97db4ed0dd1605d313ef004868502c6 \
+                    sha256  eaf0e00245fd704ca9876bf34e64807e68309c19b6b43709d92a033d566656ea \
+                    size    335981
+github.tarball_from archive
+
+depends_lib-append  port:R-dplyr \
+                    port:R-Rcpp \
+                    port:R-rlemon \
+                    port:R-tibble
+
+compilers.setup     require_fortran
+
+variant tests description "Enable testing" {
```

Comment:
it's not clear to me why this needs to be a variant - why can you not just add this to the "normal" port?

Reply:
Because of a bug in base which no one is interested to fix :) 
It considers test dependencies to be required for the build, which creates a [fake] dependency cycle with some ports.

This is the main reason. In a few other instances I made it a variant when there are ridiculously many test dependencies, some of which pull in heavy stuff which takes ages to compile.

Reply:
> Because of a bug in base which no one is interested to fix :) It considers test dependencies to be required for the build, which creates a [fake] dependency cycle with some ports.

Please cite bug URLs when you claim there is a bug. In this case it's https://trac.macports.org/ticket/69895 and we have not yet proven that there is a bug.

Reply:
@ryandesign Thank you for the link.